lib/kitchen/provisioner/chef_base.rb

```diff
@@ -183,6 +183,21 @@ def format_config_file(data)
         }.join("\n")
       end
 
+      # Generates a Hash with inherited values from the kitchen.yml configuration.
+      # These are for system options such as proxies, SSL, etc.
+      #
+      # @return [Hash] a configuration hash
+      # @api private
+      def inherited_config_rb
+        Hash.new.tap do |h|
+          h[:http_proxy] = config[:http_proxy] if config[:http_proxy]
+          h[:https_proxy] = config[:https_proxy] if config[:https_proxy]
+          h[:no_proxy] = config[:no_proxy] if config[:no_proxy]
+          h[:ftp_proxy] = config[:ftp_proxy] if config[:ftp_proxy]
+          h[:ssl_verify_mode] = config[:ssl_verify_mode].to_sym if config[:ssl_verify_mode]
```

Comment:
We should probably check the "standard" Ruby proxy environment variables here as well..